src/current/v25.2/detect-hotspots.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Detect Hotspots
+summary: Learn how to detect hotspots using real-time monitoring and historical logs in CockroachDB.
+toc: true
+---
+
+This page provides practical guidance on identifying common [hotspots]({% link {{ page.version.version }}/understand-hotspots.md %}) in CockroachDB clusters using real-time monitoring and historical logs. This tutorial assumes that you have identified a metrics outlier in your cluster. It focuses on CPU and latch contention metrics to help you identify hot-key and hot-index scenarios.
+
+## Before you begin
+
+- Review the [Understand hotspots page]({% link {{ page.version.version }}/understand-hotspots.md %}) for definitions and concepts.
+- Ensure you have access to the DB Console Metrics and the relevant logs.
+
+## Troubleshooting overview
+
+Identify potential hotspots and optimize query and schema performance. The following sections provide details for each step.
```

Comment:
suggest editing these sentence together, something like eg 

"the following sections provide detailed instructions for identifying potential hotspots and optimizing query and schema performance"

although reading through the subsections it seems like it might be more accurate to say something like

"the following sections provide detailed instructions for identifying potential hotspots and applying mitigations"

what do you think?